addons/web/static/tests/qunit.js

```diff
@@ -3,6 +3,13 @@
 
     const { Component } = owl;
 
+    // -------------------------------------------------------------------------
+    // OWL 2 Compat
+    // -------------------------------------------------------------------------
+    const { mount } = owl;
+    owl.mount = (C, target, config = {}) =>
+        mount(C, target, Object.assign({ templates: window.__ODOO_TEMPLATES__ }, config));
```

Comment:
remove this (should work, if not, fix it, and remove this)

Reply:
on it